src/__tests__/homepage.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, render, screen} from '@testing-library/react';
+import HomePage from "../pages/Home/Home.page";
+import youtubeVideos from "../mocks/youtube-videos";
+
+describe('Home page Tests', () => {
+  afterEach(cleanup);
+
+  describe('Ensure elements get rendered', () => {
+    it('Title', () => {
+      render(<HomePage />);
+      expect(screen.getByText(/YouTube video search app/, {selector: 'h1'})).toBeInTheDocument();
+    });
+  });
+
+  describe('Ensure there the info from the mock is being shown', () => {
+    it("A random video's information will be displayed if it has videoid", () => {
```

Comment:
Perhaps implementing the video cards using the `article` HTML element may allow you to use the `getByRole` with role equals to `article` and `name` equals to a random video title?